.github/workflows/pages.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         with:
           python-version: '3.10'
       - name: Install mkdocs
-        run: pip install mkdocs
+        run: pip install mkdocs==1.4.3
```

Comment:
Yeah, pip does use the latest release, but that would be a problem if someone had built their local environment with mkdocs 1.4.3, and then 2.0 was released with breaking changes... The local builds would still work but GH pages would fail because it'll get 2.0. So pinning the version here is mostly to avoid a confusing build failure down the road.